openproblems/tasks/_cell_cell_communication/_common/methods/baseline.py

```diff
@@ -1,18 +1,24 @@
 from .....tools.decorators import method
 from .....tools.utils import check_version
+from ..utils import aggregate_method_scores
 
+import functools
 import numpy as np
 import pandas as pd
 
-
-@method(
-    method_name="Random Events",
+_random_method = functools.partial(
+    method,
     paper_name="Random Events (baseline)",
     paper_url="https://openproblems.bio",
     paper_year=2022,
     code_url="https://github.com/openproblems-bio/openproblems",
     is_baseline=True,
 )
+
+
+@_random_method(
```

Comment:
Since this method is never actually used, I wouldn't decorate it (and maybe make it a private method?)

Reply:
Hmmmm, yeah I was wondering whether I should just make it private myself. However, I would actually prefer to keep them visible because I need them in their original form for the classification task that I plan to add :) 

 Also, in case someone comes up with a better way to aggregate.